modules/ocf_admin/manifests/create.pp

```diff
@@ -7,9 +7,8 @@
     '/etc/ocf-create':
       ensure => directory;
 
-    # TODO: ideally this file wouldn't be directly readable by staff
     '/etc/ocf-create/ocf-create.conf':
-      group     => ocfstaff,
+      group     => ocfroot,
```

Comment:
`approve` still needs the file to read the celery connection info; did you intend for `approve` to become root-only?

Reply:
Also, it seems the proper solution for this is to separate out `celery.broker` and `celery.backend` to a separate config file... is it worth the effort at this point when we're planning to redo secrets management for create when it gets moved into newk8s?

Reply:
I think approve is mostly used by rootstaff anyway, so this is probably a good idea?

Reply:
If you're going to make approve root-only, don't forget to update the docs. I can think of https://www.ocf.berkeley.edu/docs/staff/scripts/approve/ and https://www.ocf.berkeley.edu/docs/staff/powers/ as pages that mention this.